.cargo/config.toml

```diff
@@ -0,0 +1,18 @@
+[build]
+# default target
+target = "wasm32-wasi"
+
+[target.wasm32-wasi]
+runner = "./scripts/wasmedge-runner.sh"
+
+[env]
+# ffmpeg install path (default is ```assets/ffmpeg-lib```)
+FFMPEG_DIR = { value = "assets/ffmpeg-lib", relative = true }
+# wasi sysroot path (default is ```assets/wasi-sysroot```)
+WASI_SYSROOT = { value = "assets/wasi-sysroot", relative = true }
+# lib clang rt path (default is ```assets/clang-rt```)
+CLANG_RT = { value = "assets/clang-rt", relative = true }
+
+# config wasi-sysroot path (default is ```./asset/wasi-sysroot```)
+#BINDGEN_EXTRA_CLANG_ARGS = "--sysroot=./asset/wasi-sysroot --target=wasm32-wasi -fvisibility=default"
```

Comment:
When building the library with ```ffmpeg``` feature using cargo, users must setup the following environment variables:

* ```FFMPEG_DIR```:  the pre-built FFmpeg library path.
* ```WASI_SDK``` or  (```WASI_SYSROOT``` and ```CLANG_RT```)
* ```BINDGEN_EXTRA_CLANG_ARGS```:  set sysroot and target and function visibility for libclang.

The value for ```BINDGEN_EXTRA_CLANG_ARGS``` must use an absolute path, but the ```.cargo/config.toml``` can only set a relative path now, so I set this environment variable in ```scripts/cargo-test.sh```.

These comment lines in ```.cargo/config.toml``` can be deleted.

And I think I should add "how to build with ```ffmpeg``` feature instructions" in README and in documents.

Reply:
In ```.cargo/config.toml```, 

```toml
# config wasi-sysroot path (default is ```./asset/wasi-sysroot```)
#BINDGEN_EXTRA_CLANG_ARGS = "--sysroot=./asset/wasi-sysroot --target=wasm32-wasi -fvisibility=default"
BINDGEN_EXTRA_CLANG_ARGS = "--sysroot=/opt/wasi-sdk/share/wasi-sysroot --target=wasm32-wasi -fvisibility=default"
```
Set the ```BINDGEN_EXTRA_CLANG_ARGS``` is used for me to run with cargo directly.
Set the ```BINDGEN_EXTRA_CLANG_ARGS``` in ```scripts/cargo-test.sh``` is used to run cargo in Github actions, it can override the value in ```.cargo/config.toml```.